client/extensions/woocommerce/app/order/index.js

```diff
@@ -134,11 +152,23 @@ class Order extends Component {
 			</Button>,
 		];
 		if ( ! isEditing ) {
-			button = (
-				<Button primary onClick={ this.toggleEditing }>
+			button = [
+				<Button key="edit" primary onClick={ this.toggleEditing }>
 					{ translate( 'Edit Order' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Edit Forum Order' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).